Source/WebCore/platform/graphics/gstreamer/eme/GStreamerEMEUtilities.h

```diff
@@ -86,25 +89,28 @@ class GStreamerEMEUtilities {
 
     static const char* keySystemToUuid(const String& keySystem)
     {
-        if (isClearKeyKeySystem(keySystem))
-            return s_ClearKeyUUID;
-
         if (isUnspecifiedKeySystem(keySystem)) {
 #if USE(OPENCDM)
-            return s_WidevineUUID;
+            return s_CommonEncUUID;
```

Comment:
I have no idea why `s_WidevineUUID` was originally being used here, but switching to `CommonEncUUID` (Clearkey) doesn't make sense either, can you explain the choices here please?

Reply:
Unspecified key id is used by webm encryption. Since it's...ekhm...unspecified... WV is assumed for webm.

Reply:
I don't like the unspecified API stuff in gst, this assumption is additionally dodgy as well. I'd at least comment this, since it doesn't make any sense :-)

Reply:
I agree there's shouldn't be a place for unspecified system.
Nonetheless this change switching to Clearkey must be reverted.

Reply:
I agree a comment would be good :) Presumably this workaround breaks Clearkey webm content?

Reply:
Widevine and Playready has a specified key system. Thinking in the way incase of unspecified, clearkey can be assumed. so if unspecified key system has no place, yes this can be removed.